src/fragments/start/getting-started/reactnative/auth.mdx

```diff
@@ -0,0 +1,263 @@
+? Do you want to configure advanced settings? No, I am done.
+```
+
+To deploy the service, run the `push` command:
+
+```bash
+amplify push
+```
+
+Now, the authentication service has been deployed and you can start using it. To view the deployed services in your project at any time, go to Amplify Console by running the following command:
+
+```bash
+amplify console
+```
+
+## Create login UI
+
+Creating a login flow can be quite difficult and time consuming to get right. Luckily, Amplify UI has an [authentication](https://ui.docs.amplify.aws/react-native/connected-components/authenticator) component that provides an entire authentication flow for you, using the configuration you specified in **aws-exports.js.**
+
+### Install Amplify UI
+
+The `@aws-amplify/ui-react-native` package includes React Native specific UI components you'll use to build your app. Install it with the following command:
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+```bash
+npm install @aws-amplify/ui-react-native react-native-safe-area-context react-native-get-random-values react-native-url-polyfill
+```
+
+</Block>
+<Block name="React Native CLI">
+
+```bash
+yarn add @aws-amplify/ui-react-native react-native-safe-area-context react-native-get-random-values react-native-url-polyfill
+```
+
+You will also need to install the pod dependencies for iOS:
+
+```sh
+npx pod-install
+```
+
+</Block>
+</BlockSwitcher>
+
+### Add the Amplify UI Authenticator component
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+Open **App.js** and make the following changes:
+
+</Block>
+<Block name="React Native CLI">
+
+Open **App.tsx** and make the following changes:
+
+</Block>
+</BlockSwitcher>
+
+1. Import the `withAuthenticator` Higher-Order Component and `useAuthenticator` hook:
+
+```javascript
+
+import { withAuthenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
+
+````
+
+2. The `useAuthenticator` hook can be added to any component that needs it. Create a custom `SignOutButton` component that will be used within the App component, and destructure the `{ user, signOut }` properties returned in the context.
+
+```javascript
+
+const SignOutButton = () => {
+  const { user, signOut } = useAuthenticator((context) => [context.user]);
+  return (
+    <Pressable onPress={signOut} style={styles.buttonContainer}>
+      <Text style={styles.buttonText}>Hello, {user.username}! Click here to sign out!</Text>
+    </Pressable>
+  )
+};
+
+```
+
+3. Add the `SignOutButton` component to  your `App` component:
+
+```javascript
+
+// ...
+return (
+  <View style={styles.container}>
+    <SignOutButton />
+//...
+
+```
+
+4. Lastly, wrap your `App` export with the `withAuthenticator` Amplify UI component:
+
+```javascript
+
+export default withAuthenticator(App);
+
+```
+
+Run the app to see the new authentication flow protecting the app:
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+```bash
+npm start
+```
+
+</Block>
+<Block name="React Native CLI">
+
+```bash
+yarn start
+```
```

Comment:
Why are we telling users to run `yarn` vs `npm` based on Expo/vanllla RN usage?

Reply:
Think that having differing both `yarn` and `npm` is more confusing for beginners. Maybe just choose one and stick with it?

Reply:
changed it to only npm